src/bindings/python/src/openvino/runtime/utils/broadcasting/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.runtime import AxisSet, Node
+from openvino.runtime.utils.types import (
+    NodeInput,
+    TensorShape,
+    get_dtype,
+    make_constant_node,
+)
```

Comment:
```suggestion
```
also, pay attention that some of these imports are unused -- remove them also from `broadcasting.py`